### PR TITLE
Design cleanup for pipeline viewer

### PR DESCRIPTION
**Pull request by snide**
Hey @justinkambic. Here's a PR for some design cleanup. Thanks again for setting it up. I made the following general changes.

* Swapped out some of your custom controls for EUI native controls.
* Rewrote some of the CSS. Was able to remove a decent chunk because of the above. Rest was renaming selectors to match our guidelines. Also used the temporary "global" variables till EUI is able to be imported directly.
* Made some tweaks for smaller screens.
* Added the page level wrappers to the page and set a fixed width, which should help with scanability on those columns. We're doing similar stuff with all the new K7ish pages (a lot of management is this way now).

Dunno if I broke anything with tests. When I tried to run them they didn't seem to run. Prolly something dumb on my side, I'm not used to using tests in Kibana proper.

Anyways. Feel free to merge whenever if it looks acceptable.

### CODE REVIEW

**Comment by snide** [comment]
Oh. If possible it'd be nice to move the titling / dropdown into the page itself, rather than the header. No bigs if you're waiting, but it would help match against what we're trying to do in other places of Kibana.

image

As an example...

image

**Comment by justinkambic** [comment]
> If possible it'd be nice to move the titling / dropdown into the page itself, rather than the header.

Agreed, especially if that's the norm in the rest of the project. I think this change would be out of scope for this feature, because it would require a significant change to otherwise unrelated code. I will update https://github.com/elastic/kibana/issues/18976 to reflect this. This change definitely fits in with some of the initiatives we've got scheduled for Ingest UI.

**Comment by justinkambic** [comment]
Everything appears to be working great, reviewing code now

File: x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/config_viewer/plugin_statement.js [hunk 1]
```diff
@@ -32,28 +32,27 @@ function getProcessorStatementMetrics(processorVertex) {
...
+        warning={processorVertex.isTimeConsuming()}
         value={formatMetric(Math.round(percentOfTotalProcessorTime || 0), '0', '%', { prependSpace: false })}
       />
     ),
     (
       <Metric
         key="eventMillis"
-        className={`cv-processorMetric__eventMillis ${eventMillisHighlight}`}
+        className="configViewer__metric--eventMillis"
+        warning={processorVertex.isSlow()}
```
Comment (justinkambic) [inline x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/config_viewer/plugin_statement.js:48]: 👍

File: x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/detail_drawer.js [hunk 2]
```diff
@@ -20,7 +20,8 @@ import {
   EuiFlexGroup,
   EuiFlexItem,
   EuiButtonIcon,
-  EuiSpacer
+  EuiSpacer,
```
Comment (justinkambic) [inline x-pack/plugins/monitoring/public/components/logstash/pipeline_viewer/views/detail_drawer.js:23]: I'm fine with merging these changes to `DetailDrawer` to my PR, and they look fine to me, but I'd like @ycombinator to review this code as well before we're ready to merge to Kibana `master`.

**Review by justinkambic — APPROVED** [review]
LGTM

**Pull request merged**